src/reducers/entities/channels.ts

```diff
@@ -7,6 +7,7 @@ import {GenericAction} from 'types/actions';
 import {Channel, ChannelMembership, ChannelStats} from 'types/channels';
 import {RelationOneToMany, RelationOneToOne, IDMappedObjects, UserIDMappedObjects} from 'types/utilities';
 import {Team} from 'types/teams';
+import {isMinimumServerVersion} from 'utils/helpers';
```

Comment:
if the `sync` flag is set based on fetching or not the archived channels this is not needed here.